pandas/core/indexes/interval.py

```diff
@@ -805,7 +807,15 @@ def get_value(self, series, key):
             loc = self.get_loc(key)
         return series.iloc[loc]
 
-    @Appender(_index_shared_docs['get_indexer'] % _index_doc_kwargs)
+    @Substitution(dict(_index_doc_kwargs,
+                       **{'raises_section': textwrap.dedent("""\
+        Raises
+        ------
+        raises NotImplementedError if any method other than than the default
```

Comment:
Typically this is formatted like

```
Raises
------
ExceptionName
    Condition.
```

Reply:
Thanks @TomAugspurger - have adjusted accordingly